ctrd/container.go

```diff
@@ -500,3 +500,36 @@ func (c *Client) ResizeContainer(ctx context.Context, id string, opts types.Resi
 
 	return pack.task.Resize(ctx, uint32(opts.Height), uint32(opts.Width))
 }
+
+// WaitContainer waits until container's status is stopped.
+func (c *Client) WaitContainer(ctx context.Context, id string) (types.ContainerWaitOKBody, error) {
+	wrapperCli, err := c.Get(ctx)
+	if err != nil {
+		return types.ContainerWaitOKBody{
+			Error:      err.Error(),
+			StatusCode: -1,
+		}, fmt.Errorf("failed to get a containerd grpc client: %v", err)
+	}
```

Comment:
If we got error from `c.Get(ctx)`, we can keep the `ContainerWaitOKBody` simple and return `types.ContainerWaitOKBody{}, fmt.Errorf("failed to get a containerd grpc client: %v", err)`.

In golang, basically we don't consume the data if error != nil